File: Do zero ao Hello World/Sprint - 2/ex07.py
# Faça um Programa que leia um vetor de 5 números inteiros, mostre a soma, a multiplicação e os números

from ast import Mult
from random import randint

numeros = []
soma = 0
multiplica = 1

for i in range(5):
    numeros.append(randint(1, 9))

for n in numeros:
    soma += n
    multiplica *= n

print(numeros)
print(soma)
print(multiplica)

File: Do zero ao Hello World/Sprint - 2/ex07.go
// Faça um Programa que leia 20 números inteiros e armazene-os num vetor. Armazene os números pares no vetor PAR e os números IMPARES no vetor impar. Imprima os três vetores

package main

import (
	"fmt"
	"math/rand"
)

func main() {

	numbers := [] int{}
	sum := 0
	multi := 1

    for i := 0; i < 5; i++ {
		number := rand.Intn(9)
		numbers = append(numbers, number)
		sum += number
		multi *= number
	}

	fmt.Println("Números: ", numbers)
	fmt.Println("Soma   : ", sum)
	fmt.Println("Multip.: ", multi)
}
